test/samePxth.test.ts

```diff
@@ -0,0 +1,13 @@
+import { createPxth, samePxth } from '../src';
+
+describe('samePxth', () => {
+    it('should compare 2 paths correctly', () => {
+        expect(samePxth(createPxth([]), createPxth([]))).toBeTruthy();
+        expect(
+            samePxth(createPxth(['a', 'b']), createPxth(['a', 'b'])),
+        ).toBeTruthy();
+        expect(
+            samePxth(createPxth(['a', 'b']), createPxth(['b', 'a'])),
+        ).toBeFalsy();
+    });

```

Comment:
Test should contain more "edge" cases: comparing empty path with '' path, comparing paths with spaces / dots, etc.